CONTRIBUTING.md

```diff
@@ -8,7 +8,6 @@ First of all, thank you for contributing to MeiliSearch! The goal of this docume
 - [How to Contribute](#how-to-contribute)
 - [Development Workflow](#development-workflow)
 - [Git Guidelines](#git-guidelines)
-- [Release Process (for internal team only)](#release-process-for-internal-team-only)
```

Comment:
Is this wanted? Just to be sure. I don't know if it is written somewhere in which case my bad